src/app/pages/global-reach/global-reach.js

```diff
@@ -31,82 +18,49 @@ const BaseClass = componentType(spec, canonicalLinkMixin, loaderMixin);
-        const map = new mapboxgl.Map({
-            container: 'mapdiv',
-            style: mapbox.style,
-            center: mapCenter,
-            zoom: 2
-        });
-
-        map.scrollZoom.disable();
-        map.dragPan.disable();
-        map.doubleClickZoom.disable();
-        map.on('load', () => {
-            if (map.loaded()) {
-                map.resize();
-                this.model.loaded = 'loaded';
-                this.update();
-                this.hideLoader();
-            }
-        });
-
-        const mapObject = {
-            mapObj: map,
-            model
-        };
-
-        return new Map1(mapObject);
-    }
-
     onDataLoaded() {
         const data = this.pageData;
-        const mapPromise = this.createMap({
-            pageType: 'landing',
-            heading: data.title,
-            buttonText: data.header_text
-        });
 
-        mapPromise.then((map) => {
-            this.regions.map.append(map);
-            this.regions.stat.attach(new State({
-                model: data.section_1_cards.map((obj) => (
-                    {
-                        imageUrl: obj.image.image,
-                        uText1: obj.number,
-                        uText2: obj.unit,
-                        lowerText: obj.description
-                    }
-                ))
-            }));
-            this.regions.studentinfo.attach(new Studentinfo({
-                model: {
-                    header1: data.section_2_header_1,
-                    description1: data.section_2_blurb_1,
-                    linkUrl1: data.section_2_link_1,
-                    linkText1: data.section_2_cta_1,
-                    image1: data.section_2_image_1_url,
-                    header2: data.section_2_header_2,
-                    description2: data.section_2_blurb_2,
-                    linkUrl2: data.section_2_link_2,
-                    linkText2: data.section_2_cta_2,
-                    image2: data.section_2_image_2_url
-                }
-            }));
-            this.regions.schoolmap.attach(new Schoolmap({
-                model: {
-                    heading: data.section_3_heading,
-                    description: data.section_3_blurb,
-                    linkText: data.section_3_cta,
-                    linkUrl: data.section_3_link
+        this.regions.self.attach(new Map1({
+            model: {
+                pageType: 'landing',
+                heading: data.title,
+                buttonText: data.header_text
+            }
+        }));
+        this.regions.self.append(new State({
+            model: data.section_1_cards.map((obj) => (
+                {
+                    imageUrl: obj.image.image,
+                    uText1: obj.number,
+                    uText2: obj.unit,
+                    lowerText: obj.description
                 }
-            }));
-        });
+            ))
+        }));
+        this.regions.self.append(new Studentinfo({
```

Comment:
This model can use documentation for the fields.

Reply:
It wasn't clear to me how this was much different from any other section, but I gave it a shot.